app/Http/Controllers/User/BlogController.php

```diff
@@ -53,6 +49,65 @@ public function store(Request $request)
         $blog->title = $request->title;
         $blog->content = $request->content;
         $blog->save();
-        return redirect()->back();
+
+        return redirect()->back()->with('message', 'Create new blog success !');
+    }
+
+    /**
+     * Delete one blog.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id) 
+    {
+        if ($this->defineUser()->can_delete === Blog::IS_TRUE) {
+            $blog = Blog::find($id);
+            if (isset($blog))
+                $blog->delete();
+            else 
+                return redirect()->back()->withErrors('Delete blog error');
+        }
+
+        return redirect()->back()->with('message', 'Delete blog success !');
+    }
+
+    /**
+     * Edit view blog.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $blog = Blog::find($id);
+        if (isset($blog)) 
+            return view('blog/edit', ['blog' => $blog]);
+        else
+            return redirect()->back()->withErrors('Blog not found');
+    }
+
+    /**
+     * Update one blog.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request ,$id)
+    {
+        $request->validate([
+            'title' => 'required',
+            'content' => 'required',
+        ]);
+        $blog = Blog::find($id);
+        if (isset($blog)) {
+        $blog->title = $request->title;
+        $blog->content = $request->content;
+        $blog->save();
+        } else {
```

Comment:
Please apply code, for same block.
if (true) { blog-> save; return ....}
return failed;

Reply:
I fixed